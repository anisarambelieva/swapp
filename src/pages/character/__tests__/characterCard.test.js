import { render } from '@testing-library/react';

import CharacterCard from '../characterCard.js';

describe('<CharacterCard />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <CharacterCard
        name="testname"
        imageSrc="test imageSrc"
        height="test height"
        weight="test weight"
        species="test species"
        homeWorld="test home world"
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
