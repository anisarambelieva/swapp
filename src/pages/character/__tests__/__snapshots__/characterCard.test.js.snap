// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CharacterCard /> should match snapshot 1`] = `
.c3 {
  color: #4E5B6E;
  margin: 0;
  font-weight: bold;
}

.c4 {
  color: #4BD5EE;
}

.c2 {
  height: 300px;
  object-fit: cover;
  width: 100%;
  margin-bottom: 5px;
}

.c0 {
  background-color: #FFF;
  padding: 10px 30px;
  margin-left: 15px;
  border-radius: 5px;
}

.c1 {
  text-align: center;
  color: #4BD5EE;
  font-weight: bold;
}

<div>
  <div
    class="c0"
  >
    <h4
      class="c1"
    >
      testname
    </h4>
    <img
      class="c2"
      src="test imageSrc"
    />
    <p
      class="c3"
    >
      Height
      : 
      <span
        class="c4"
      >
        test height
      </span>
    </p>
    <p
      class="c3"
    >
      Weight
      : 
      <span
        class="c4"
      >
        test weight
      </span>
    </p>
    <p
      class="c3"
    >
      Species
      : 
      <span
        class="c4"
      >
        test species
      </span>
    </p>
    <p
      class="c3"
    >
      Home World
      : 
      <span
        class="c4"
      >
        test home world
      </span>
    </p>
  </div>
</div>
`;
